### PR TITLE
Debug settings enhancement

### PR DESCRIPTION
**Pull request by Honny1**
This PR enhances the class `DebugSettings` that contains settings for debugging.

### CODE REVIEW

File: tests/unit_tests/test_debug_settings.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    )),
+    ([
+        "BUTTON-SHOW-ALL-RULES-AND-OVAL-TEST-DETAILS"
+    ], DebugSetting(
+        include_debug_script=True,
+        button_show_all_rules_and_oval_test_details=True
+    )),
+    ([], DebugSetting()),
+])
+def test_update_settings_with_debug_flags(debug_flags, expected_values):
```
Comment (evgenyz) [inline tests/unit_tests/test_debug_settings.py:93]: Blank line missed

**Comment by evgenyz** [comment]
Will manpage auto-generation work with this setup?

**Comment by Honny1** [comment]
@evgenyz  Auto-generation of manpage works fine with this setup.

File: tests/unit_tests/test_debug_settings.py (hunk 1 above)
Comment (Honny1) [reply tests/unit_tests/test_debug_settings.py:126]: I think it's not necessary. It's the end of a decorator.

**Review by evgenyz — APPROVED** [review]

**Pull request merged**